### DISCUSSION TITLE
DoubleCloud

### DISCUSSION THREAD

**Discussion by denisovval** [aquasecurity/trivy#5329] [Adopters]
### Please select all the scan targets that you are using

- [X] Container Images
- [X] Filesystem
- [X] Git Repository
- [ ] Virtual Machine Images
- [X] Kubernetes
- [ ] AWS

### Which scanners are you using on those scan targets?

- [X] OS packages and software dependencies in use (SBOM)
- [X] Known vulnerabilities (CVEs)
- [X] IaC issues and misconfigurations
- [X] Sensitive information and secrets
- [ ] Software licenses

### Additional Information

We use trivy inside CI/CD pipelines, Kubernetes clusters, and GithubActions to protect the whole infrastructure. And all reports are aggregated within DefectDojo
https://trust.double.cloud/

**Comment by AnaisUrlichs** [comment]
Amazing! Thank you so much for sharing 🥳
